app/cypress/e2e/SecvisogramPage/FormEditorTab.cy.js

```diff
@@ -607,4 +607,268 @@ describe('SecvisogramPage / FormEditor Tab', function () {
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // generate button should do nothing without revision history entries
+      cy.get(
+        '[data-testid="document-tracking-initial_release_date-generateButton"]'
+      )
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // create new revision history item
+      cy.get(
+        '[data-testid="menu_entry-/document/tracking/revision_history-add_item_button"]'
+      ).click({ force: true })
+      cy.get(
+        '[data-testid="menu_entry-/document/tracking/revision_history/0"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-revision_history-0-date"] input[type="date"]'
+      )
+        .type('2020-01-01')
+      cy.get(
+        '[data-testid="attribute-document-tracking-revision_history-0-date"] input[type="time"]'
+      )
+        .type('13:41')
+      cy.get(
+        '[data-testid="attribute-document-tracking-revision_history-0-number"] input'
+      )
+        .clear()
+        .type('1.0.0')
+
+      // current release should still be empty
+      cy.get('[data-testid="document/tracking-fieldButton"]').click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // generate button should enter correct date and time
+      cy.get(
+        '[data-testid="document-tracking-initial_release_date-generateButton"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="date"]'
+      ).should('have.value', '2020-01-01')
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="time"]'
+      ).should('have.value', '13:41')
+
+      // just regenerating should be still the same
+      cy.get(
+        '[data-testid="document-tracking-initial_release_date-generateButton"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="date"]'
+      ).should('have.value', '2020-01-01')
+      cy.get(
+        '[data-testid="attribute-document-tracking-initial_release_date"] input[type="time"]'
+      ).should('have.value', '13:41')
+    })
+
+    it('fill release dates with more than one revision history entry', function () {
+      cy.visit('?tab=EDITOR')
+
```

Comment:
Please also add a version `0.9.0` (2019-12-31T12:01:00Z) here to check that the right date is selected